packages/storage/src/database/global/metadata-database.ts

```diff
@@ -96,4 +99,26 @@ export class ModelMetadataDatabase extends ModelBasic<
   public async count(filter?: Filter<TMetadataDatabase>) {
     return await this.collection.countDocuments(filter);
   }
+
+  public static async init(session?: ClientSession) {
+    const collection = ModelCollection.getInstance<TMetadataDatabase>(
+      zkDatabaseConstant.globalProofDatabase,
+      DB.proof,
+      zkDatabaseConstant.globalCollection.proof
+    );
+    if (!(await collection.isExist())) {
+      /*
+        databaseName: string;
+        databaseOwner: string;
+        merkleHeight: number;
+        appPublicKey: string;
+      */
+      await collection.index({ databaseName: 1 }, { unique: true, session });
+      await collection.index({ databaseOwner: 1 }, { unique: false, session });
+      await collection.index({ merkleHeight: 1 }, { unique: false, session });
```

Comment:
I think we need to keep it, since user need to query database that have specific height. We can also compound with databaseName, databaseOwner, merkleHeight as well